test/Generators/Microsoft.Gen.Metrics/Unit/ParserTests.Diagnostics.cs

```diff
@@ -96,27 +96,6 @@ class Transitive
         Assert.Contains("Test.Transitive ⇆ Test.Interim", diag.GetMessage());
     }
 
-    [Fact]
-    public async Task StrongTypeCounter_CyclicReference_Guid()
```

Comment:
This test is not relevant anymore - we don't consider `Guid.Empty` since it's `static`